### ISSUE TITLE
[Legitimate Site Blocked] www.blockchain-company.io

### ISSUE THREAD

**Issue by rmourey26** [MetaMask/eth-phishing-detect#32900]
The domain and website in question is: https://www.blockchain-company.io/

This is one of my company websites. 

Deployed using NextJS Stripe Supabase subscription starter template. 

Heres the repo:

github.com/theblockchaincompany/subscription-starter

This didn't occur yesterday when I visited the site but occurred today about 30 minutes ago using Chrome browser with metamask enabled extension on a Chromebook.

Very curious about the logic used by eth-phishing that caused this.

If you need any other details from me, I am happy to provide, Thanks

**Comment by AlexHerman1** [comment]
hi, what is this website? its not clear what the content has to do with the URL or with "blockchains"

**Comment by rmourey26** [comment]
Its the homepage associated with my self owned development company The Blockchain Company.io LLC. Github home page github.com/theblockchaincompany

I am a software engineer who builds web2 and web3 based applications. 

The main page is currently in flux and could use additional pages, metadata etc.

A while back I decided to use stripe vercel and nextjs to manage my website/products and simply didn't complete it for a variety of reasons. 

A few months ago, I forked Uniswap's UI for a client. I also created a separate demo (Uniswap UI) that's sitting at defiprotocol.blockchain-company,io. 

I wouldn't be surprised that if that also had something to do with the issue here although I literally just now logged into my wallet using the metamask extension at defiprotocol.blockchain-company.io and same Chromebook device.
